app/views/conversations/index.haml

```diff
@@ -1,41 +1,36 @@
-- content_for :head do
-  = javascript_include_tag :inbox
-
 - content_for :page_title do
-  = t('.conversations_inbox')
+  = t(".conversations_inbox")
 
-.container-fluid#conversations_container
+.container-fluid#conversations-container
   .row
     .col-md-4
       .sidebar#left_pane
         .sidebar-header.clearfix#left_pane_header
           .pull-right
-            = link_to t(".new_conversation"), conversations_path, class: "btn btn-default"
+            = link_to t(".new_conversation"), conversations_path, class: "new-conversation-btn btn btn-default"
           %h3
             = t(".inbox")
 
-        .conversation-inbox#conversation_inbox
-          .stream.conversations
+        .conversation-inbox#conversation-inbox
+          .conversations-form-container.stream.conversations
             - if @visibilities.count > 0
               = render partial: "conversations/conversation", collection: @visibilities, as: :visibility
             - else
               .no-conversations
-                = t('.no_messages')
+                = t(".no_messages")
             .pagination-container
               = will_paginate @visibilities, previous_label: "&laquo;", next_label: "&raquo;", inner_window: 1,
                   renderer: WillPaginate::ActionView::BootstrapLinkRenderer
-
-
     .col-md-8
-      - if @conversation
-        .stream_container
-          #conversation_show
+      .conversations-form-container.stream_container
+        #conversation-show{class: @conversation ? "" : "hidden"}
+          - if @conversation
             = render 'conversations/show', conversation: @conversation
-      - else
-        .stream_container.hidden
-          #conversation_show
-        .framed-content.clearfix#conversation_new
-          .new-conversation
-            %h3.text-center
-              = t("conversations.index.new_conversation")
-            = render "conversations/new"
+        #conversation-new{class: @conversation ? "framed-content clearfix hidden" : "framed-content clearfix"}
+          - if no_contacts
+            .well.text-center= t(".no_contacts")
```

Comment:
Ah yes, I understand the mistake. I forgot a function when I wrote this PR that checks the recipient list is not empty from client side. I will rename `javascripts.conversation.create.no_contacts` to `javascripts.conversations.create.no_recipient` and `conversations.create.no_contacts` to `conversations.create.no_recipiant` to avoid confusion with `conversations.new.no_contacts`.